addon/components/rdf-input-fields/currency-input.js

```diff
@@ -0,0 +1,17 @@
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+import SimpleInputFieldComponent from '@lblod/ember-submission-form-fields/components/rdf-input-fields/simple-value-input-field';
+
+export default class RdfInputFieldsCurrencyInputComponent extends SimpleInputFieldComponent {
+  inputId = 'input-' + guidFor(this);
+
+  currencyIsocode = 'EUR';
+
+  @action
+  updateValue(e) {
+    if (e && typeof e.preventDefault === 'function') e.preventDefault();
+    this.value = e.target.value.trim();
```

Comment:
[inputmask also adds a `inputmask` property to the target which has an ](https://robinherbots.github.io/Inputmask/#/documentation#unmaskedvalue)`unmaskedvalue()` method. I'm not sure if we want the mask to be stored in the DB?

Reply:
I think you should store the mask in the database.. We will get a data model like this... and the consumer of the data can do everything with this (calculations, viewing, sorting,..). But the thing here is as we can only define ONE sh:path on a field we can only do the string input value with that. We can of course just add the other predicates to the same subject in the ember component itself. Do we want this, is this allowed in ESFF? 
```ttl
# Currency Input
<http://ember-submission-form-fields/source-node>
  <http://mu.semte.ch/vocabularies/ext/currencyInputValue> "0,0" ;
  <http://schema.org/currency> "EUR" ;
  <http://schema.org/price> "0,0 EUR" ;
  <http://schema.org/amount> 0.0 .
```

=> the reason why we did not save the number already is that we have to convert the incoming and outgoing value each time. as the input-mask cannot handle a number `123456.00` to be displayed as `123456,00` as we have set the radix point to a comma and not the dot. It will be shown as `12345600,00`. Another use case that the currency,price and amount predicates will be handy for.

Reply:
Will just save it as a decimal and convert the value each time